src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,38 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = (props) => {
+  const {
+    title,
+    body,
+    comments,
+    user,
+  } = props.post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {`Posted by ${user.username}`}
+
+          <UserInfo user={user} />
+        </p>
```

Comment:
it seems that there should not be `username` [screen](https://prnt.sc/7nJMd7WZrI-z)